pep-0696.rst

```diff
@@ -220,17 +303,17 @@ Subclassing
 Subclasses of ``Generic``\ s with ``TypeVarLike``\ s that have defaults
 behave similarly to ``Generic`` ``TypeAlias``\ es.
 
-.. code:: py
+.. code-block:: py
 
    class SubclassMe(Generic[T, DefaultStrT]): ...
 
    class Bar(SubclassMe[int, DefaultStrT]): ...
    reveal_type(Bar())        # type is Bar[str]
    reveal_type(Bar[bool]())  # type is Bar[bool]
 
-   class Foo(SubclassMe[int]): ...
+   class Foo(SubclassMe[float]): ...
 
-   reveal_type(Foo())  # type is <subclass of SubclassMe[int, int]>
+   reveal_type(Foo())  # type is <subclass of SubclassMe[int, float]>
```

Comment:
Shouldn't this be `SubclassMe[float, str]`? Also, the reveal_type() output would just be "type is Foo".

Reply:
Yeah you're correct on both points. The `<subclass of SubclassMe[int, str]>` is just to show `DefaultStrT` is constrained to `str` in `SubclassMe`. To avoid this I could actually use `DefaultStrT` as an attribute but I think this might be a little harder to follow. Do you have any other ideas as to how else this could be shown?